x/cheqd/types/tx_msg_deactivate_did_payload.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+var _ IdentityMsg = &MsgDeactivateDidPayload{}
+
+func (msg *MsgDeactivateDidPayload) GetSignBytes() []byte {
+	return ModuleCdc.MustMarshal(msg)
+}
+
+func (msg *MsgDeactivateDidPayload) ToDid() Did {
+	return Did{
+		Id: msg.Id,
+	}
+}
+
+// Validation
+
+func (msg MsgDeactivateDidPayload) Validate(allowedNamespaces []string) error {
+	return msg.ToDid().Validate(allowedNamespaces)
```

Comment:
```suggestion
	return validation.ValidateStruct(&msg,
		validation.Field(&msg.Id, validation.Required, IsDID(allowedNamespaces)),
	        )
```
+ add `validation "github.com/go-ozzo/ozzo-validation/v4"` to `import`
+ remove `ToDid` method